### PR TITLE
add back due dates

### PR DESCRIPTION
**Pull request by futurepaul**

### CODE REVIEW

File: src/components/EventCreator.tsx [hunk 1]
```diff
@@ -36,21 +36,43 @@ export function EventCreator(props: { onSave: () => Promise<void> }) {
...
+        throw new Error("Outcomes must be unique");
+      }
+
+      let event_maturity = f.event_maturity;
+
+      // If there's no event_maturity set, default to one minute from now
+      if (!event_maturity) {
+        console.log("no event maturity");
+        event_maturity = new Date(Date.now() + 60 * 1000).toISOString();
+      }
```
Comment (benthecarman) [inline src/components/EventCreator.tsx:58]: think we should just error if none set

Comment (futurepaul) [reply src/components/EventCreator.tsx:58]: if you pick a date it's gonna be set to midnight, so you have to wait a whole day to do an observe

Comment (benthecarman) [reply src/components/EventCreator.tsx:58]: if we do a minute from now then it'll immediately expire and they won't be able to do anything, so we should at least pick a better default

Comment (futurepaul) [reply src/components/EventCreator.tsx:58]: oh good point okay I'll go back to required

**Review by benthecarman — APPROVED** [review]

**Pull request merged**